### PR TITLE
feature/IISCRUM-2543

### PR DESCRIPTION
**Pull request by timyllym**
Send "FoldTabRow" query param when opening link

### CODE REVIEW

File: dist/panel-triggers/module.js [hunk 1]
```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
         };
         _this.onLinkIconClick = function (event, url) {
             event.stopPropagation();
-            window.top.location.href = url;
+            // Add 'foldTabRow' query param so that Grafana tab row panel keeps folded
+            var urlBase = url.substring(0, url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var urlQuery = url.substring(url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var queryObj = _this.parseParamsStringToObject(urlQuery);
```
Comment (sapajune) [inline dist/panel-triggers/module.js]: I've seen this kind of three code lines repeated alot recently. Should there be a common function for parsing URL into an object that provides nice structure for accessing URL parts? I would also imagine there already being some library for that.

File: dist/panel-triggers/module.js [hunk 2]
```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
...
+            queryObj['foldTabRow'] = 'true';
+            var newQuery = _this.parseParamsObjectToString(queryObj);
```
Comment (sapajune) [inline dist/panel-triggers/module.js]: Technically converting object to string would be formatting not parsing. 😄 

This function could also be implemented as member of URL object mentioned above (e.g. url.getQueryString()).

File: dist/panel-triggers/module.js [hunk 3]
```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
         };
         _this.onLinkIconClick = function (event, url) {
             event.stopPropagation();
-            window.top.location.href = url;
+            // Add 'foldTabRow' query param so that Grafana tab row panel keeps folded
+            var urlBase = url.substring(0, url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var urlQuery = url.substring(url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var queryObj = _this.parseParamsStringToObject(urlQuery);
+            queryObj['foldTabRow'] = 'true';
```
Comment (sapajune) [inline dist/panel-triggers/module.js]: Do we have use case for multiple tab rows in single dashboard? If so, simple boolean won't be sufficent.

**Review by sapajune — APPROVED** [review]
Looks good.

See comments.

File: dist/panel-triggers/module.js (hunk 1 above)
Comment (timyllym) [reply dist/panel-triggers/module.js]: I'll find a JS library for url parsing from npm

File: dist/panel-triggers/module.js (hunk 2 above)
Comment (timyllym) [reply dist/panel-triggers/module.js]: Yep

File: dist/panel-triggers/module.js (hunk 3 above)
Comment (timyllym) [reply dist/panel-triggers/module.js]: We can have multiple tab rows in dashboard and now this affects them all. The problem is how to identify which of the tab rows is in question. We could add a user editable ID field to Tab Row panel, but then the correct ID would have to be added to all existing Tab Rows in all dashboards/tenants that need to adapt to this behavior.

Comment (sapajune) [reply dist/panel-triggers/module.js]: To support multiple tab rows the multiple parameters should be supported and maybe for current use value "all" to indicate it applies to all tab rows. How does that sound?

Comment (timyllym) [reply dist/panel-triggers/module.js]: Ok, lets use keyword "all" now for starters

**Review by bulkware — APPROVED** [review]

**Pull request merged**